evap/staff/templates/staff_course_email.html

```diff
@@ -31,13 +33,16 @@
                         {% elif field.name == "body" %}
                             <div class="form-label {% if field.errors %}error{% endif %}" name="{{ field.name }}">{{ field.label }}</div>
                             <div class="form-field"><textarea class="form-control {% if field.errors %}error{% endif %}" cols="40" rows="10" id="{{ field.id_for_label }}" name="{{ field.name }}" oninput="changedFieldValue('{{ field.name }}');"></textarea></div>
+                        {% elif field.name == "addresses" and addresses %}
+                            <div class="form-label " name="{{ field.name }}">{{ field.label }}</div>
+                            <div class="form-field"><input class="form-control {% if field.errors %}error{% endif %}" id="{{ field.id_for_label }}" name="{{ field.name }}" type="text" value="{{ addresses }}"></div>
```

Comment:
if i understand it correctly, you're not really using that `addresses` form field for user input, and even the data you render here comes as a additional template parameter not coupled to the form field, right? if yes, could you remove that field and render the data outside the form? @janno42 suggested to just put it into a message (search for `messages.info` in staff/views.py), one email address per line.